lib/gelf/severity.rb

```diff
@@ -29,9 +34,11 @@ module Levels
   # Maps Ruby Logger levels to syslog levels as is.
   DIRECT_MAPPING = {DEBUG   => 7, # Debug
                     INFO    => 6, # Info
-                    # skip 5 Notice
+                    NOTICE  => 5, # Notice
                     WARN    => 4, # Warning
                     ERROR   => 3, # Error
+                    CRIT    => 2, # Critical
                     FATAL   => 2, # Critical
```

Comment:
It's intentionally redundant, so that gelf-rb clients dealing with syslog data and the DIRECT_MAPPING format can use a intuitive placeholder that represents the Syslog "Critical" level, rather than needing to remember "in gelf-rb you use FATAL to mean 'Critical' when when dealing with syslog levels".

I can't remember why I chose "CRIT" instead of "CRITICAL" when I wrote this over 4 years ago.  Looking at it now, I think I would choose CRITICAL.

I'm not sure who else is using DIRECT_MAPPING and what the use case is for mapping Ruby errors levels like FATAL in that way (when _wouldn't_ Ruby levels use LOGGER_MAPPER?), so anyhow I left the FATAL->2 (Critical) mapping in there too, for backwards compatibility.